test/errors.js

```diff
@@ -1167,5 +1192,49 @@ ${M}      </pre>
         },
       );
     });
+
+    it('negative: oneof', async () => {
+      await assertErrorFree(`
+        <emu-grammar type="definition">
+          Foo :: one of \`a\` \`b\` \`c\`
+        </emu-grammar>
+
+        <emu-clause id="sec-example" type="sdo">
+        <h1>Static Semantics: Example</h1>
+        <dl class='header'></dl>
+          <emu-grammar>
+            Foo :: one of \`a\` \`b\` \`c\`
+          </emu-grammar>
+          <emu-alg>
+            1. Return *true*.
+          </emu-alg>
+        </emu-clause>
+      `);
+    });
+
+    it('negative: oneof (subset)', async () => {
+      await assertErrorFree(`
+        <emu-grammar type="definition">
+          Foo :: one of \`a\` \`b\` \`c\`
+        </emu-grammar>
+
+        <emu-clause id="sec-example" type="sdo">
+        <h1>Static Semantics: Example</h1>
+        <dl class='header'></dl>
+          <emu-grammar>
+            Foo :: one of \`a\`
```

Comment:
This should read

```suggestion
            Foo :: \`a\`
```

We should prohibit "one of" with only one alternative.

Reply:
Do you mean that
```
<emu-gramar>
  Foo ::
    `a`
    `b`
    `c`
    `d`
</emu-grammar>
<emu-clause>
  1. Return 1.
</emu-clause>
```
is preferable to
```
<emu-gramar>
  Foo :: one of `a` `b` `c` `d`
</emu-grammar>
<emu-clause>
  1. Return 1.
</emu-clause>
```
?

Reply:
For completeness sake, the case that triggered this PR would be
```
      <emu-grammar>
        TerminalDigit :: one of
          `A` `B` `C` `D` `E` `F` `G` `H` `I` `J` `K` `L` `M` `N` `O` `P` `Q` `R`
          `S` `T` `U` `V` `W` `X` `Y` `Z` `a` `b` `c` `d` `e` `f`
      </emu-grammar>
      <emu-alg>
        1. Let _digit_ be the source text matched by this Parse Node.
        1. ...
```

Reply:
🤔 okay I'm convinced.

Reply:
I changed the test to ``Foo :: one of `a` `b` `` instead. Allowing ``Foo :: `a` `` should probably be done in a separate PR since it requires changing `rhsMatches` to compare different kinds of productions. Also disallowing ``Foo :: one of `a` `` should probably be done in the linter in a separate PR.